package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "model-instagram-people-service",
+  "version": "1.0.0",
+  "description": "Handle People-related requests for Model-Instagram clone",
+  "main": "./server/index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "start:server": "export NODE_ENV=production && nodemon --max_old_space_size=4000 ./server/index.js",
```

Comment:
Suggest changing this to development and creating a separate production environment later